README.md

```diff
@@ -30,6 +30,17 @@ repositories {
 }
 
 apply plugin: 'com.uber.okbuck'
+
```

Comment:
This should not be needed by default. Please remove this blockk

Reply:
I have removed the block. I was under impression that it would throw error and stop buck work if I remove it. I will check it in next pull request and provide more context by creating an issue if I reproduce it. For now, let' get the command line updated.